backend/app/api/ai.py

```
@@ -0,0 +1,138 @@
+from fastapi import APIRouter, HTTPException, UploadFile, File
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)